source/vocab/relations.ttl

```diff
@@ -424,11 +420,16 @@ prefix relsubtype: <http://id.loc.gov/vocabulary/preservation/relationshipSubTyp
     rdfs:subPropertyOf :accompanies;
     owl:inverseOf :supplement .
 
-:hasSeries a owl:ObjectProperty;
-    rdfs:label "har serie"@sv;
+:partOfSeries a owl:ObjectProperty;
```

Comment:
`.partOfSeries` is missing the corresponding `owl:inverseOf :seriesOf;` which in turn probably is best to change from `:hasSeries` to `:partOfSeries`.